test/cluster/mzcompose.py

```diff
@@ -57,10 +57,10 @@
     Kafka(),
     SchemaRegistry(),
     Localstack(),
-    Clusterd(name="clusterd1"),
-    Clusterd(name="clusterd2"),
-    Clusterd(name="clusterd3"),
-    Clusterd(name="clusterd4"),
+    Clusterd(name="clusterd1", workers=2),
+    Clusterd(name="clusterd2", workers=2),
+    Clusterd(name="clusterd3", workers=2),
+    Clusterd(name="clusterd4", workers=2),
```

Comment:
I used 2 as the default worker count because that seemed to be what's used by the most tests. Not opposed to using 1 instead and changing the tests with overwrites.